### ISSUE TITLE
Specific channel launch option

### ISSUE THREAD

**Issue by chirimbolito** [vinszent/gnome-twitch#168]
I couldn't find any way to launch the program in a specific channel. It would be a nice add.

**Comment by vinszent** [comment]
Do you mean like straight from the command line? Something like this:

    gnome-twitch --open-channel summit1g

**Comment by chirimbolito** [comment]
Exactly, couldn't find anything about that. I instantly moved from livestreamer when I found this app and was kinda used to run some channels in that way using other software/scripts (like the gnome twitch panel applet). In this case it would work way better cause you can always browse channels without the need to close the app if you get bored.

**Comment by vinszent** [comment]
Ok sure no problem
